### ISSUE TITLE
Boilerplate does not run

### ISSUE THREAD

**Issue by kokokenada** [DAB0mB/ReactNativeMeteorBoilerplate#2]
I followed the instructions in the readme.

When I run, I get:

````
Loading dependency graph, done.
error: bundling: UnableToResolveError: Unable to resolve module `react-addons-pure-render-mixin` from `/Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules/react-meteor-data/lib/createContainer.js`: Module does not exist in the module map or in these directories:
  /Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules
,   /Users/kenono/node_modules

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset packager cache: `rm -fr $TMPDIR/react-*` or `npm start --reset-cache`.
    at UnableToResolveError (/Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:488:5)
    at p.catch.error (/Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:366:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
Bundling `index.android.js`  99.5% (411/412), failed.
````

I was able to get a bit further by changing my package.json to:
````
  "dependencies": {
    "meteor-node-stubs": "^0.2.6",
    "react": "16.0.0-alpha.6",
    "react-addons-pure-render-mixin": "^15.5.2",
    "react-meteor-data": "^0.2.10",
    "react-native": "0.44.0"
  },
````
But then I get a big red box on the phone:
````
Objects are not valid as a React child (found object with keys....
````

I also tried putting react-native back to 0.43.3, same problem...

**Comment by DAB0mB** [comment]
@kokokenada Are you trying to run this on iOS? Did you try to reset the cache? Try running:

    $ npm start -- --reset-cache

**Comment by kokokenada** [comment]
Yes.  I cleared the cache (blew away node-modules, and the various other things suggested.) I take it, you couldn't replicate the problem?  Somehow I managed to get further, but now I'm blocked on a different problem.  Will open a new issue.

**Comment by kokokenada** [comment]
Just reran the process from scratch and I get:
````
Loading dependency graph, done.
error: bundling: UnableToResolveError: Unable to resolve module `react-addons-pure-render-mixin` from `/Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules/react-meteor-data/lib/createContainer.js`: Module does not exist in the module map or in these directories:
  /Users/kenono/apps/ReactNativeMeteorBoilerplate/node_modules
,   /Users/kenono/node_modules

````
running
````
npm install --save react-addons-pure-render-mixin
````
fixed it.
